### ISSUE TITLE
ArcBlock - Due diligence

### ISSUE THREAD

**Issue by icochecker** [ico-check/ico-check#37]
# [ArcBlock](https://www.arcblock.io/)

## Team
- Robert Mao (Ceo/) - 2017 - Present | [Linkedin profile](https://www.linkedin.com/in/robertmao)
- Flavien Charlon (Chief Scientist) - 2017 - Present | [Linkedin profile](https://www.linkedin.com/in/flaviencharlon)
- Jean Chen (VP of Marketing) - Aug 2017 - Present | [Linkedin profile](https://www.linkedin.com/in/jeancheninchina)

## Team members with no mention of ArcBlock in their Linkedin profiles:
- Joe Wallin (Legal Counsel) - Sep 2015 - Present | [Linkedin profile](https://www.linkedin.com/in/wallin) **does not mention ArcBlock**

### Advisors
- Ding Lei
- Slman Dhanani | [Linkedin profile](https://www.linkedin.com/in/saldhanani/)
- Justin Tomboulian | [Linkedin profile](https://www.linkedin.com/in/justin-tomboulian-69b61919/)
- Mike Cartwright | [Linkedin profile](https://www.linkedin.com/in/cartwrightx/)
- Lawrence Lerner | [Linkedin profile](https://www.linkedin.com/in/lawrencelerner/)
- Jiaxi Lin | [Linkedin profile](https://www.linkedin.com/in/jiaxi-lin-a536133/)
- Huishen Cui | [Linkedin profile](https://www.linkedin.com/in/huishen-cui-7254b362/)
- Maggie Wu | [Linkedin profile](https://www.linkedin.com/in/maggie-wu-164a0797/)
- Violin Wang | [Linkedin profile](https://www.linkedin.com/in/violin-wang-b81a6673/)
- Andrew Le

### ARCBLOCK, INC.
**File Number:** 6636015         
**Incorporation Date / Formation Date:** 28 Nov 2017        
**Entity Kind:** Corporation           
**Entity Type:** General          
**Residency:** Domestic         
**State:** DELAWARE    

![arcblock-delaware](https://user-images.githubusercontent.com/32852887/35463297-3b21ecc0-02f0-11e8-9145-29bdec408f23.png)

### ARCBLOCK FOUNDATION LTD
![arcblock-singapore](https://user-images.githubusercontent.com/32852887/35463686-2eed4542-02f2-11e8-955a-0861db12b2a1.png)

**Registered Address:**     
9, TEMASEK BOULEVARD       
04-02    
SUNTEC TOWER TWO       
SINGAPORE 038989               

**UEN:** 201803249Z

**Company Type Description:** EXEMPT PRIVATE COMPANY LIMITED BY SHARES      
**Incorporation Date:**     
**Primary SSIC Code:**           
**Status:**         
**Industry:** 

- [Source: SG UEN Search](https://www.uen.gov.sg/)  

- ArcBlock Foundation Limited, a Delaware corporation???    
- ArcBlock Foundation,  Limited, a company limited by guarantee to be registered in Hong Kong???      

## Smart contract
- [Contract on github](https://github.com/ArcBlock/ArcBlock-Token/blob/master/ArcBlockTokenFlattened.sol)

## Github
- [ArcBlock](https://github.com/arcblock)

## Product
- No working product

## Vesting
- No vesting schedule for Team tokens
![arcblock-vesting-1](https://user-images.githubusercontent.com/32852887/34871851-553a08fc-f78f-11e7-821e-a7937ecf82dd.png)
![arcblock-vesting-2](https://user-images.githubusercontent.com/32852887/34871854-571fb266-f78f-11e7-8e30-2c4a79f30389.png)

### Token distribution
![distribution](https://user-images.githubusercontent.com/32852887/35464000-e58c693a-02f3-11e8-828b-9fa4c2fb8306.png)

### Token allocation

# Users banned for asking legitimate questions. See comments below

**Comment by jeffshek** [comment]
GitHub - It's light on details.

https://github.com/ArcBlock

**Comment by globehopstera** [comment]
Banned from ArcBlock Telegram Group for asking about ArcBlock prototype.

I was just kicked off and banned from the main ArcBlock Telegram Group for asking about the prototype, without warning. I was not attempting to spread FUD but was inquiring about the project as part of due diligence. 
When asked about the prototype, the ArcBlock admin gave a vague response such as "Related information will be released soon." When I sought clarification, I was booted from the Telegram group.
This kind of action by the Admin of an ICO is obviously a serious Red Flag.
It is worth noting that the ArcBlock Roadmap states for Q4 2017 "Complete the prototype for the first end-to-end application built on top of ArcBlock." and for Q2 2017 "Proof of concept and prototype development of Blocklet with Ethereum."  So, they do claim they have a prototype(s) but when asked directly to provide details they are evasive or worse.  
In conclusion, while these are red flags, they do not necessarily mean this project is a scam. It could still be legitimate. Perhaps the admin is just a noob or went rogue. Nevertheless, this and all ICOs should conduct themselves in a professional manner and treat prospective investors respectfully. ICO must understand investor due diligence and adequately address questions/concerns.  In this way, all parties will be justly rewarded.

**Comment by AeroTimber** [comment]
Banned from ArcBlock Telegram Group for asking the contract address of the ABT token on Ethplorer.

Out of the blue I was kicked and banned from the ArcBlock Telegram Group just because I very politely asked for the contract address of the ArcBlock  ABT token on Ethplorer because I couldnt find it and wanted to add a custom token to my MEW wallet. 

I made it clear I was not asking for the ETH adress to send my contribution for the public sale.  I was told that my MEW wallet would accept the ABT token. When I explained that i knew that and just asked for the contract address for the token I was kicked and banned. Asking for the reason of the ban remained unanswered. 
Very unprofessional behavior by an admin of an ICO. At least they could have explained they didnt know or that it would be given later. Being banned for asking  ' normal' questions is not a good sign for the professionalism of this team. Ultimately they depend on customers and treating us with respect is the least they can do!

They should take lessons from the admins of the very professionally led INS ICO.
